Dockerfile

```diff
@@ -59,13 +59,13 @@ RUN apk add --no-cache bind-tools docker
 #####################################
 FROM node:16.15.0-alpine
 
-ENV DOCKER_COMPOSE_VERSION 1.25.5
+ENV DOCKER_COMPOSE_VERSION v2.5.0
```

Comment:
Why does the version before didn't have the `v` prefix, but now it does?

Reply:
Because they have changed the nomenclature https://github.com/docker/compose/releases/tag/v2.5.1 and https://github.com/docker/compose/releases/tag/1.25.5